packages/react-components/src/templates/DashboardPageBody.tsx

```diff
@@ -22,34 +22,34 @@ import {
   PastEventsDashboardCard,
   RecentSharedOutputs,
 } from '../organisms';
-import { perRem } from '../pixels';
+import { rem } from '../pixels';
 import { Card, Paragraph, Link, Headline2 } from '../atoms';
 import { lead } from '..';
 import { Accordion } from '../molecules';
 import { confidentialIcon, giftIcon, learnIcon } from '../icons';
 
 const styles = css({
   display: 'grid',
-  gridRowGap: `${56 / perRem}em`,
-  marginBottom: `${24 / perRem}em`,
+  gridRowGap: `${rem(56)}em`,
+  marginBottom: `${rem(25)}em`,
 });
 
 const containerStyles = css({
-  marginTop: `${24 / perRem}em`,
+  marginTop: `${rem(24)}em`,
 });
 
 const listStyles = css({
-  paddingLeft: `${18 / perRem}em`,
+  paddingLeft: `${rem(18)}em`,
 });
 
 const infoStyles = css({
   color: lead.rgb,
-  padding: `${3 / perRem}em 0 ${24 / perRem}em`,
-  lineHeight: `${24 / perRem} em`,
+  padding: `${rem(3)}em 0 ${rem(24)}em`,
```

Comment:
Isn't this substitution supposed to be 

```suggestion
  padding: `${rem(3)} 0 ${rem(24)}`,
```


The spacing looks a bit weird in here https://2092.hub.asap.science/, I think it may be due to this change

![2092 hub asap science_(iPhone XR)](https://user-images.githubusercontent.com/16595804/193323092-09fa13e7-67cf-4f53-8d45-83ba5963d7a1.png)

Reply:
good catch, fixed!